SPECS/ding-libs/ding-libs.spec

```diff
@@ -88,7 +89,7 @@ Utility functions to manipulate filesystem pathnames
 %ldconfig_scriptlets -n libpath_utils
 
 %files -n libpath_utils
-%license COPYING COPYING.LESSER
+%doc COPYING COPYING.LESSER
```

Comment:
Aren't these licenses? I'm wondering why they were flipped to `%doc` instead of `%license`.

Reply:
I can do a blame on this change and report back.